Dockerfile

```diff
@@ -8,9 +8,9 @@ RUN adduser --system --shell /sbin/nologin --home /opt/azafea-metrics-proxy azaf
 USER azafea
 WORKDIR /opt/azafea-metrics-proxy/src
 
-COPY Pipfile.lock .
+COPY Pipfile Pipfile.lock .
 
-RUN pipenv install --ignore-pipfile --dev
+RUN pipenv install --dev
```

Comment:
What you really want to do is call `pipenv sync`. Then you still only need `Pipfile.lock`. I thought I made that change here, but I [only did it in Azafea](https://github.com/endlessm/azafea/commit/0bb4705801e05ff73fa295e9133db934dc46610c).

Reply:
That's what I did in the first version of this patch. If you read the diff in https://github.com/pypa/pipenv/pull/6098/files they changed https://pipenv.pypa.io/en/latest/workflows.html from recommending `pipenv sync` to `pipenv install`. I think both would work. OK, I'll go back to v1.

Reply:
Ah, I didn't read the earlier context. That is... quite the choice they made there. How did all those people involved in the project review that PR and decide that changing the behavior of `install` again was OK?

IMO, I think it's better to stick with `sync` since it clearly does what we want with both older and newer versions of pipenv.

Reply:
We should probably start pinning the version of `pipenv` used since clearly the maintainers have little regard for compatibility. I also like the "we're using semantic versioning now but the major version is just the year" policy. Uh, ok.